### DISCUSSION TITLE
PROJQUAY Bug Tracker no longer public?

### DISCUSSION THREAD

**Discussion by ReLehmann** [quay/quay#1532] [General]
Hi all,

I was about to suggest a feature request as a registered user, but discovered that you are no longer allowed to create issues in the JIRA project. Is this conscious?

P.S. Feature: Support of SCIM standard for automated provisioning of user accounts. 

Kind regards,
Lenny

**Comment by BillDett** [comment]
Hi Lenny- sorry about that, definitely not intentional. We've had some internal changes to our Jira project over the past few months, perhaps this got lost in the shuffle. Let me do some investigation here.

Bill

**Comment by BillDett** [comment]
Lenny- we might have found the issue- can you please try again?

**Reply by ReLehmann** [reply]
Hi Bill, works again. Thank you!! 🥇
